src/main/java/org/stellar/sdk/MemoId.java

```diff
@@ -37,4 +37,9 @@ public boolean equals(Object o) {
     MemoId memoId = (MemoId) o;
     return id == memoId.id;
   }
+
+  @Override
+  public String toString() {
+    return Long.toString(id);
```

Comment:
`id` is actually supposed to be interpreted as unsigned. Can you change this to:

```
import com.google.common.primitives.UnsignedLongs;
...
  @Override
  public String toString() {
    return UnsignedLongs.toString(this.id);
  }
```

Reply:
Sure, fixed